packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -2943,6 +2945,7 @@ class MaterialBasedCupertinoThemeData extends CupertinoThemeData {
     CupertinoTextThemeData? textTheme,
     Color? barBackgroundColor,
     Color? scaffoldBackgroundColor,
+    Color? selectionHandleColor,
```

Comment:
We should also update the docs in `TextSelectionThemeData` to reflect the changes here.

https://github.com/flutter/flutter/blob/296df33be866d0cd8387524e288d9b280e725795/packages/flutter/lib/src/material/text_selection_theme.dart#L49-L58

Reply:
Done!